flyteidl/protos/flyteidl/core/artifact_id.proto

```diff
@@ -20,18 +20,38 @@ message ArtifactKey {
 message ArtifactBindingData {
   // These two fields are only relevant in the partition value case
   oneof partition_data {
-    string partition_key = 2;
-    bool bind_to_time_partition = 3;
+    string partition_key = 1;
```

Comment:
Safe to change these?

Reply:
yeah, we're dropping all tables on this one.